### PR TITLE
[README.md] Enhance linking and grammar

### PR DESCRIPTION
**Pull request by fvsamson**
- Fix grammar and enhance wording.
- Consecutive web-links without non-linked, non-whitespace characters are … problematic; thus rectified.
- References to outdated specifications do not make much sense, hence omitted, especially as the link to the current specification immediately followed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,9 +10,8 @@
 
 # csaf_distribution
 
-An implementation of a [CSAF](https://csaf.io/)
-[2.0 Spec](https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html)
-([Errata](https://docs.oasis-open.org/csaf/csaf/v2.0/csaf-v2.0.html))
+Implements a [CSAF](https://csaf.io/)
+([Specification v2.0](https://docs.oasis-open.org/csaf/csaf/v2.0/csaf-v2.0.html))
```
Comment (bernhardreiter) [inline README.md:14]: Hi @fvsamson, nice to read from you! :)

The problem here is that https://docs.oasis-open.org/csaf/csaf/v2.0/csaf-v2.0.html only links to the errata and there is no easily found link to the current main content of the 2.0 speficiation. It is there, but under appendix A of the errata. Also the new link to the main specification https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html, which you have deleted in this change, does not link to the errata. So your change makes it much harder to access both. 

Any idea to resolve this for good? What about making a new section that explains the standard and the link both with more explanation?

**Review by bernhardreiter — CHANGES_REQUESTED** [review]
The "An implementation of a" -> "Implements a" is an improvement, but now a link to the main body of the standard is missing. (See comment).

File: README.md (hunk 1 above)
Comment (fvsamson) [reply README.md]: Fixed IMO.  Please re-review.

**Review by tschmidtb51 — APPROVED** [review]
LGTM

**Review by bernhardreiter — APPROVED** [review]
Thx

**Pull request merged**